src/extensions/menu.extension.ts

```diff
@@ -17,8 +20,32 @@ export class Menu {
     options?: {
       level?: number;
       headline?: string;
+      showHelp?: boolean;
```

Comment:
It might be better to allow caching explicitly, so it would be good to add a setCache property and check it later before changing the cache.

Reply:
Added!